src/PDO/Peachpie.Library.PDO/PDOStatement.cs

```diff
@@ -243,31 +263,259 @@ public bool bindColumn(PhpValue colum, ref PhpValue param, int? type = default(i
         }
 
         /// <inheritDoc />
-        public bool bindParam(PhpValue parameter, ref PhpValue variable, int data_type = 2, int? length = default(int?), PhpValue? driver_options = default(PhpValue?))
+        public bool bindParam(PhpValue parameter, PhpAlias variable, PDO.PARAM data_type = PDO.PARAM.PARAM_STR, int length = default(int), PhpValue driver_options = default(PhpValue))
```

Comment:
`default(int)` is `0`, is it alright?

Reply:
It is not used anywhere at the moment, as the Statement option is not implemented so far. However agreed that -1 is a better option.